packages/container/src/components/FieldPluginContainer.tsx

```diff
@@ -311,7 +313,25 @@ export const FieldPluginContainer: FunctionComponent = () => {
         <AccordionSummary>
           <Typography variant="h3">Content</Typography>
         </AccordionSummary>
-        <AccordionDetails sx={{ position: 'relative' }}>
+        <AccordionDetails
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            gap: '2rem',
+            position: 'relative',
+          }}
+        >
+          <Container
+            sx={{
+              display: 'flex',
+              p: { xs: 0 },
+            }}
+          >
+            <LanguageView
+              language={language}
+              setLanguage={setLanguage}
+            />
```

Comment:
Sorry for being picky, and not specifying this before: I believe the `ContentView` is the more commonly used than the language: therefore the `LanguageView` should be placed below the `ContentView`.